loom-cli/main/loom.hs

```diff
@@ -137,8 +138,33 @@ main = do
             Exit.exitWith ec
       Watch port sf ->
         watch port sf
-      Paths (PathsCommand _ _ _) ->
-        IO.hPutStrLn IO.stderr "TODO"
+      Paths (PathsCommand bundle sets _grouping) -> do
+        cwd <- getCurrentDirectory
+        config <- orDie renderLoomConfigTomlError $ resolveConfig cwd
+        loom <- resolveLoom config
+        let allResolved =
+              pure (loomResolvedConfig loom) <> (loomResolvedConfigs loom)
+            -- TODO: Component dirs
+            -- TODO: resolvePurescript?Get FilePaths from LoomFile originals
+            -- Hell; can we go *backwards* to infer a glob? or is it better to hack it in
```

Comment:
I wouldn't go backwards. I would vote for changing to a glob if anything, but I'm also ok with putting in a separate glob just for this command to get it to work first, and then work backwards to replacing the current find logic. Totally up to you of you course.

Reply:
Good idea; will do.